libs/effects/Effect/System.idr

```diff
@@ -14,18 +15,23 @@ data System : Effect where
      Time : sig System Integer
      GetEnv : String -> sig System (Maybe String)
      CSystem : String -> sig System Int
+     Usleep : (i : Int) ->
+              {auto prf : So (i >= fromInteger 0 && Delay (i <= fromInteger 1000000))} ->
```

Comment:
I just copied the type from `System.usleep`. `fromInteger` is probably not necessary, it's shown when you type `:t usleep` in REPL, but it's not present in [the source file](https://github.com/idris-lang/Idris-dev/blob/master/libs/base/System.idr). However, `auto` is used in that source file. Should I still remove it?